src/design-tokens/tier-2-usage/colors-text.json

```diff
@@ -49,6 +49,11 @@
           },
           "disabled": {
             "value": "{eds.color.neutral.300}"
+          },
+          "brand": {
+            "primary": {
+              "value": "{eds.color.brand.grape.700}"
```

Comment:
text color brand added for this case, thought it would be broader use so added in tier2, lmk if should be tier3